packages/ui/src/components/collectibles/native/collectible-text.native.tsx

```diff
@@ -1,4 +1,12 @@
 import { useEffect, useState } from 'react';
+import WebView from 'react-native-webview';
+
+// fetchInscriptionTextContent is a simple axios get
+// maybe its better to use axios directly in UI?
+// better yet would be to add the text content to the collectible object
+// in the service and then we can use that here
+// i tried that but it was not working as expected
+import { fetchInscriptionTextContent } from '@leather.io/query';
```

Comment:
I added the `'@leather.io/query'` package to the UI library but I'm not sure if that's a good fit. 

I want to use `fetchInscriptionTextContent` so I:
- if it's a text inscription
- I can fetch the text

`fetchInscriptionTextContent` is actually a simple `axios` call:
```
export async function fetchInscriptionTextContent(src: string) {
  const res = await axios.get(src, { responseType: 'text' });
  return res.data;
}
```

I tried making this call again client side but it got into a real mess in the `FlashList` `renderItem`. I tried to append this to the collectibles service but again it got quite messy. 

We will improve that service soon and when we do I believe the text content should be provided in the collectible query response by the service

Reply:
cc @tigranpetrossian but IMO it's a mistake to add network dependencies to the `ui` package. Would prefer inversion of control / dependency injection instead.